lte/gateway/python/magma/subscriberdb/crypto/milenage.py

```diff
@@ -45,14 +43,15 @@ def generate_eutran_vector(self, key, opc, sqn, plmn):
         sqn_bytes = bytearray.fromhex('{:012x}'.format(sqn))
         rand = Milenage.generate_rand()
 
+
         mac_a, _ = Milenage.f1(key, sqn_bytes, rand, opc, self.amf)
         xres, ak = Milenage.f2_f5(key, rand, opc)
         ck = Milenage.f3(key, rand, opc)
         ik = Milenage.f4(key, rand, opc)
 
         autn = Milenage.generate_autn(sqn_bytes, ak, mac_a, self.amf)
         kasme = Milenage.generate_kasme(ck, ik, plmn, sqn_bytes, ak)
-        return rand, xres, autn, kasme
+        return rand, xres, autn, kasme, ck, ik
```

Comment:
there should not be any changes to `generate_eutran_vector`

Reply:
I've restored generate_eutran_vector.